src/offline.js

```diff
@@ -0,0 +1,16 @@
+const PouchDB = require('pouchdb')
+
+export function init (cozy, offline) {
+  const options = offline.options || {}
+  const docTypes = offline.docTypes || []
+  for (let docType of docTypes) {
+    addDocType(cozy, docType, options)
+  }
+}
+
+export function addDocType (cozy, docType, options) {
+  if (cozy._offline_databases === false) {
```

Comment:
We may check if it is an array, is there another way to check that like `if (!Array.isArray(cozy._offline_databases) { cozy._offline_databases = [] }`